website/content/labs/lab8.md

```diff
@@ -132,12 +132,16 @@ On a separate terminal, you can also try running a few Docker commands:
 ## Task 2: Git Submodules
 [20 points]
 
-Your `cs220-redis-example` repository has two directories:
+Your `cs220-redis-example` repository contains code that extends the Redis server with some additional commands. We will only be concerned with one of these commands: the `EXAMPLE.HGETSET` command. This commands operates on a hash table and takes two parameters: a key and a value. It will fetch the value currently associated with the given key, and will replace its value with the one provided as a parameter.
 
-* `module/` - This directory contains a Redis module that adds a few extra commands. We will be using just one of them: EXAMPLE.HGETSET, which does an HGET operation followed by an HSET operation. In other words, it gets the value of a key in a hash table (which will be returned by the command), and then sets the value of that key to a new value. Note: this example module is the same example module provided by Redis.
-* `tests/` - This directory contains a simple program that uses the hiredis library to connect to a Redis server to test the HGETSET operation.
+TODO: Include and example
```

Comment:
Was this intentional?

Reply:
Yes, the module/ and tests/ description were moved to a different point in the writeup. My latest commit adds an example here.